features/cryptocell/FEATURE_CRYPTOCELL310/trng.c

```diff
@@ -22,9 +22,10 @@
 
 #include <string.h>
 #include "trng_api.h"
+#include "mbedtls/platform.h"
 
-extern CRYS_RND_State_t     rndState;
-extern CRYS_RND_WorkBuff_t  rndWorkBuff;
+extern mbedtls_platform_context  ctx;
+static CRYS_RND_WorkBuff_t  rndWorkBuff;
```

Comment:
Should `rndWorkBuff` be part of mbedtls_platform_context?

Reply:
As mentioned in https://github.com/ARMmbed/mbed-os/pull/7099#discussion_r213934349, this is only a sandbox for the initialization operation, and not used later, so I don't think it should be part of the context